modules/fbx/fbx_document.cpp

```diff
@@ -2108,6 +2113,15 @@ Error FBXDocument::_parse(Ref<FBXState> p_state, String p_path, Ref<FileAccess>
 		WARN_PRINT(vformat("FBX: ignored %d further ufbx warnings", ignored_warning_count));
 	}
 
+	document_extensions.clear();
```

Comment:
Are we sure it's ok to clear extensions?

Reply:
yes absolutely since this is the list of USED extensions for the current import and not the list of all extensions